jlm/llvm/opt/RvsdgTreePrinter.cpp

```diff
@@ -115,6 +118,78 @@ RvsdgTreePrinter::AnnotateNumRvsdgNodes(
   annotateRegion(*rvsdg.root());
 }
 
+void
+RvsdgTreePrinter::AnnotateNumMemoryStateInputsOutputs(
+    const rvsdg::graph & rvsdg,
+    util::AnnotationMap & annotationMap)
+{
+  static std::string_view argumentLabel("NumMemoryStateTypeArguments");
+  static std::string_view resultLabel("NumMemoryStateTypeResults");
+  static std::string_view inputLabel("NumMemoryStateTypeInputs");
+  static std::string_view outputLabel("NumMemoryStateTypeOutputs");
```

Comment:
I checked compiler explorer, and it is better to leave these as non-static. It does not do a strlen at runtime either way, but making them static adds an extra level of indirection to access the length and string data.

Reply:
I removed that `static`s. Could you approve again, please?